part_three.asm

```diff
@@ -0,0 +1,22 @@
+# COMP3410 Program Template
+# Author: Edward Prins
+# Assignment: Programming Assignment #1, Part 3
+# Due Date: 11 February 2015
+
+.text
+	
+	li $t0, 1		#
+	li $t1, 0		#
+	li $t3, 15		#
+	add $t1, $t0, $t1	#
+
+loop:	addi $t0, $t0, 2	#
+	add $t1, $t0, $t1	#
+	bne $t0, $t3, loop	#
```

Comment:
-6 Instructions require partial sum to be stored and retrieved from memory on each iteration